core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/InternalBucketMetricValue.java

```diff
@@ -68,10 +72,16 @@ public String getWriteableName() {
     }
 
     @Override
+    public double getValue() {
+        return value();
+    }
```

Comment:
I only added this because similar single value interfaces like Sum/Min/Max etc... implement `double getValue()`  in addition to `double value()`, just to stay consistend. Happy to remove this from this interface, as you said we don't really need it.